components/list/demo/resposive.md

```diff
@@ -2,16 +2,16 @@
 order: 5
 title:
   zh-CN: 响应式的栅格列表 
-  en-US: Grid
+  en-US: Responsive grid list
 ---
 
 ## zh-CN
 
-响应式的栅格列表。
+响应式的栅格列表。尺寸与 [Layout Grid](https://ant.design/components/grid-cn/#Col) 保持一致。
 
 ## en-US
 
-Responsive Grid List.
+Responsive grid list. The size property as same as [Layout Grid](https://ant.design/components/grid/#Col).
```

Comment:
Responsive grid list. The size property is the same as [Layout Grid](https://ant.design/components/grid/#Col).